tv_apps/smart-system/js/permission_manager.js

```diff
@@ -160,15 +181,11 @@
       if (this.message.classList.contains('hidden')) {
         this.message.classList.remove('hidden');
       }
-      if (!this.moreInfoBox.classList.contains('hidden')) {
-        this.moreInfoBox.classList.add('hidden');
-      }
       this.devices.innerHTML = '';
       if (!this.deviceSelector.classList.contains('hidden')) {
         this.deviceSelector.classList.add('hidden');
       }
       this.buttons.dataset.items = 2;
-      this.no.style.display = 'inline';
```

Comment:
As discussed, this don't resolve the problem of queued dialogs doesn't reset `remember.checked`. 
It's an existing problem.I think we can open another bug to fix it.